<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateScientistWindow</class>
 <widget class="QDialog" name="UpdateScientistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logos/logos/SACR_Icon_red.png</normaloff>:/logos/logos/SACR_Icon_red.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update Scientist</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabelName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabelGender">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="genderFemaleButton">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="genderMaleButton">
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="genderOtherButton">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="yearBornLabel">
       <property name="text">
        <string>Year Born:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabelBorn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="yearBornInput"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="yearDiedLabel">
       <property name="text">
        <string>Year Died:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabelDeath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="yearDiedInput">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Leave empty if still alive</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="ScientistsInfoLable">
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="ScientistsAddInfo">
     <property name="placeholderText">
      <string>Feel free to use HTML in this box.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="inputInfoUrlLabel">
     <property name="text">
      <string>Add link to more information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputInfoUrl"/>
   </item>
   <item>
    <widget class="QPushButton" name="ScientistAddPic">
     <property name="text">
      <string>Update picture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
